tests/core/editable/keystrokes/delbackspacequirks/manual/nbspdeleting.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: 4.16.2, bug, range, 3819
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, sourcearea
+
+1. Put caret right after the word `Hello`.
+2. Type a space. The caret is now between two visual spaces.
+3. Press the left arrow to put caret before two visual spaces.
+4. Use the delete key to remove one space from the right side.
```

Comment:
Since the `delete` key on MacBooks works like `backspace` on Windows, please add a note that on mac you should press `fn+delete` to spare googling for testers.